recipes/entt/3.x.x/conanfile.py

```diff
@@ -38,8 +42,15 @@ def configure(self):
                 "%s requires a compiler that supports at least C++%s" % (self.name, minimal_cpp_standard))
             return
 
-        version = tools.Version(self.settings.compiler.version)
-        if version < minimal_version[compiler]:
+        # Compare versions asuming minor satisfies if not explicitly set
+        def gte_compiler_version(version1, version2):
+            v1, *v1_res = version1.split(".", 1)
```

Comment:
I don't like this implementation as it may lead into infinite loops. there is a `Version` class for that.

Reply:
What kind of input could lead to infinite loop?

Reply:
The `Version` class will consider `minor` equals to `0` if it is not set, which is exactly the opposite of what we are doing here. The expression `tools.Version(version1) >= tools.Version(version2)` doesn't work the same.

About infinite recursion... it would require infinite `.` in both versions